servicemanager/servicemanager.go

```diff
@@ -57,6 +58,7 @@ const tmpRootFSDir = "tmprootfs"
 // ServiceStorage provides API to create, remove or access services DB.
 type ServiceStorage interface {
 	GetService(serviceID string) (ServiceInfo, error)
+	GetAllLatestServices() ([]ServiceInfo, error)
```

Comment:
I agree that `GetAllServices` doesn't cover what it exactly returns. But `GetAllLatestServices` sounds ambiguous, what about `GetLatestVersionServices`?

Reply:
use this name `GetLatestVersionServices `